composeApp/build.gradle.kts

```diff
@@ -100,3 +100,9 @@ android {
         debugImplementation(compose.uiTooling)
     }
 }
+dependencies {
+    implementation(libs.androidx.runtime.saveable.android)
+    implementation(libs.androidx.runtime.livedata)
+    implementation(libs.androidx.lifecycle.livedata.core.ktx)
+    implementation(libs.androidx.ui.android)
+}

```

Comment:
Why are you importing all these dependencies? They're also put in the wrong spot of the build file.